service/topology_state_machine.cc

```diff
@@ -17,17 +17,11 @@ namespace service {
 logging::logger tsmlogger("topology_state_machine");
 
 const std::pair<const raft::server_id, replica_state>* topology::find(raft::server_id id) const {
```

Comment:
The function is explicitly defined to return nodes in non left state.

Reply:
Right.